galaxy.yml

```diff
@@ -28,13 +29,15 @@ authors:
   - "Sean O'Keeffe <seanokeeffe797@gmail.com>"
   - "furhouse <furhouse@users.noreply.github.com>"
 version: "0.1.0"
-license: "GPL-3.0-or-later"
+license:
+  - "GPL-3.0-or-later"
 tags:
   - foreman
   - katello
   - satellite
   - orcharhino
 readme: "README.md"
+homepage: "https://theforeman.org/"
```

Comment:
This looks a bit generic, but there might not be a better (more specific) landing page atm.

Reply:
Yeah, agreed. The docs link already goes to the "plugin" subpage, so that felt redundant too.